style.css

```diff
@@ -62,6 +51,83 @@ h1 {
   z-index: 10;
 }
 
+/* To-Do List Skeleton: */
+.list-container {
+  background: black;
+  width: 40%;
+  top: 30%;
+  left: 30%;
+  right: 50%;
+  min-height: 100px;
+  min-width: 300px;
+  border-radius: 0px;
+  z-index: 2;
+  position: relative;
+  margin: 0;  
+  margin-top: 50px;
+}
+
+/* List input button */
+.list-input {
+  position: relative;
+  padding: 20px;
+
+}
+
+.list-input input{
+ width: 75%;
+ height: 30px;
+ padding: 10px;
+ color: #FF00FF;
+ position: relative;
+ border-radius: 0px;
+ font-family:'Gochi Hand', 'Just Another Hand', cursive;
+ font-size: 16px;
+ border-style: solid;
+ border-color: #FF00FF;
+ background-color: black;
+ border-width: 2px;
+ letter-spacing: 0.1em;
+}
+
+.list-input input:focus{
```

Comment:
Great use in focus selector to pick out the element in that focus area 

great work ⭐